HackYourFuture-CPH/FP-annotatetheweb-class10
# Functioning homepage

When typing url and pressing enter:

- new project will be created
- new screenshot will be created to that project
- screenshot_key is saved in context. 

To test: 
Write something in the 'insert url' input field.
If it worked, in the console you will get two messages: 

- first is coming from the context file, it will say 'from context initial context value' (because the initial value is: initial context value)
- second is a screenshot_key

The screenshot_key is saved to the context, so next time you write and send something in the url input field (do not refresh the page! ), you should get two messages again in console:

- first is from context file, now it should say 'from context {screenshotkcode} '
- and a new screenshot_id

The page is not redirecting now to projects, so we can test it. 

Mobile version functionality is not set up. Issue should be created for that.
Picture should be fixed at somepoint, because it's not very pretty.
